src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechSynthesis.java

```diff
@@ -462,57 +467,58 @@ public String publishSpeechRequested(String toSpeak) {
    * @param block
    * @return
    */
-  public List<AudioData> parse(String toSpeak, boolean block){
-    
+  public List<AudioData> parse(String toSpeak, boolean block) {
+
     // TODO - not sure if we want to support this notation
     // but at the moment it seems useful 
     // splitting on sound effects ...
     // TODO - use SSML speech synthesis markup language
-    
+
     // broadcast the original text to be processed/parsed
     invoke("publishSpeechRequested", toSpeak);
-    
+
     // normalize to lower case
     toSpeak = toSpeak.toLowerCase();
-    
+
     // process substitutions
-    for (String substitute: substitutions.keySet()) {
+    for (String substitute : substitutions.keySet()) {
       toSpeak = toSpeak.replace(substitute, substitutions.get(substitute));
     }
 
     List<String> spokenParts = parseEffects(toSpeak);
-    
+
     toSpeak = filterText(toSpeak);
-    
+
     // we generate a list of audio data to play to support
     // synthesizing this speech
     List<AudioData> playList = new ArrayList<AudioData>();
-   
+
     for (String speak : spokenParts) {
-  
+
       AudioData audioData = null;
       if (speak.startsWith("#") && speak.endsWith("#")) {
-        audioData = new AudioData(System.getProperty("user.dir") + File.separator + "audioFile" + File.separator + "voiceEffects" + File.separator + speak.substring(1, speak.length() - 1) + ".mp3");
+        audioData = new AudioData(
+            System.getProperty("user.dir") + File.separator + "audioFile" + File.separator + "voiceEffects" + File.separator + speak.substring(1, speak.length() - 1) + ".mp3");
       } else {
         audioData = new AudioData(getLocalFileName(speak));
       }
-      
+
       if (speak.trim().length() == 0) {
         continue;
       }
-      
+
       process(audioData, speak, block);
-      
+
       // effect files are handled differently from generated audio
       playList.add(audioData);
     }
     // FIXME - in theory "speaking" means generating audio from some text
     // so starting speaking event is when the first audio is "started"
     // and finished speaking is when the last audio is finished
-  
+
     return playList;
   }
-  
+
   /**
```

Comment:
Seems like a lot of whitespace modification .. is it because there are spaces on the originals or is this cr/lf mods ?

Reply:
yup spaces on the originals